ert3/config/_parameters_config.py

```diff
@@ -76,6 +76,27 @@ def _ensure_lower_upper(cls, values):  # type: ignore
         )
 
 
+class _LogUniformInput(_ParametersConfig):
+    lower_bound: float
+    upper_bound: float
+
+    @root_validator
+    def _ensure_lower_upper(cls, values):  # type: ignore
```

Comment:
Could we use
`def _ensure_lower_upper(cls, values: Dict[str, Any]) -> Dict[str, Any]:`
?

Reply:
yes, we should. Will fix in all other classes too.